internal/commands/app/create_inputs.go

```diff
@@ -174,7 +174,16 @@ func (i *createInputs) resolveClusters(ui terminal.UI, client atlas.Client, grou
 
 		var clusterName string
 		if len(i.Clusters) == 1 {
-			clusterName = i.Clusters[0]
+			for _, c := range clusters {
+				if c.Name == i.Clusters[0] {
+					clusterName = i.Clusters[0]
+				}
+			}
+			if clusterName == "" {
+				return nil, nil, fmt.Errorf("could not find atlas cluster '%s'", i.Clusters[0])
+			}
+		} else if len(clusters) == 1 {
```

Comment:
do we have a test case for this already?  can we add one if not?

Reply:
I've grouped these template app creation tests and added two new test cases:

1. For this comment: for whether a cluster will be automatically selected if user does not supply cluster and there is only one option returned from the atlas clusters endpoint
2. For whether a prompt appears if user does not supply cluster and there are multiple options returned from the atlas clusters endpoint